src/scenes/Wallet/WalletManage.tsx

```diff
@@ -1,32 +1,40 @@
 import React, { useState } from 'react'
-import CreateTransaction from './CreateTransaction'
-import ReviewTransactions from './ReviewTransactions';
+import WalletFund from './WalletFund';
+import CreateTransaction from '../Transaction/CreateTransaction'
+import ReviewTransactions from '../Transaction/ReviewTransactions';
 
-function TransactionManagement() {
+function WalletManage() {
 
   const steps = [
     {
       name: 'create',
+      title: 'Create Transaction',
       component: <CreateTransaction />,
     },
     {
       name: 'review',
+      title: 'Review Transactions',
       component: <ReviewTransactions />
     },
+    {
+      name: 'fund',
+      title: 'Fund Wallet',
+      component: <WalletFund />
+    },
   ]
   const [defaultTab, setDefaultTab] = useState<'create' | 'review' | 'execute'>('create');
 
   return (
     <div className='TransactionManagement container card shadow my-5 p-5'>
         <h1 className='text-center mb-3'>
-                Manage Transactions
+                Manage Wallet
```

Comment:
Any particular reason for having this big indentation?

Reply:
Fixed in this PR: https://github.com/5afe/safe-space/commit/1c583b14a7ef33f9b8ed5512d131658d4ca431ae